src/aind_data_schema_models/models/process_names.csv

```diff
@@ -15,6 +15,8 @@ Image background subtraction
 Image cell classification
 Image cell quantification
 Image cell segmentation
+Image cross-round alignment
+Image cross-session alignment
```

Comment:
what's the difference between cross-round and cross-session? My guess is that cross-round is on the HCR side and cross-session is on the ophys session. But do they really need distinct names - it's the same concept?

Reply:
@saskiad I think you're right. Just discuss this with the team since I am not very familiar with ophys, the only difference is that cross-round is for 3D images of the same tissue for round to round registration and cross-session is for 2D image alignment. Would Image cross-image alignment be a better option?

Reply:
Hi @saskiad, please let me know what you think. Thanks!